### DISCUSSION TITLE
sqlite3 versioning for local ChromaDB

### DISCUSSION THREAD

**Discussion by JeremyGarman** [open-webui/open-webui#3412] [General]
So I'm running the Open-WebUI installation on a hardened RHEL 9 instance (hardened using CIS Server Level 2 controls), within a venv instance.  I was able to successfully install ollama and can run llama3 CLI, and was able to install node.js and python3.11 for a local installation, which also worked...  but... when I try to run Open-WebUI (using CLI command:  open-webui serve), I get an error that the local ChromaDB instance requires sqlite3 >= 3.35.  I'm currently running 3.34.  I verified with sqlite.org that they released 3.46 awhile ago, but downloading the source code and compiling my own instance blew up my OS.  Further, Red Hat only supports up to 3.34 as current, and I have no timeline when they will be releasing 3.35.

Does anyone have a better process to compile a local instance, or have any alternatives to using sqlite3 3.35 for ChromaDB?

**Comment by mdidomenico4** [comment]
similar issue https://github.com/open-webui/open-webui/discussions/5190
